CMakeLists.txt

```diff
@@ -175,12 +181,15 @@ list(APPEND fms_c_src_files
   mosaic/mosaic_util.c
   mosaic/read_mosaic.c
   mpp/mpp_memuse.c
+  diag_manager/diag_yaml.c
+  parser/yaml_parser_binding.c
 )
 
 # Collect FMS header files
 list(APPEND fms_header_files
   include/file_version.h
   include/fms_platform.h
+  diag_manager/diag_yaml.h
```

Comment:
@thomas-robinson  Dont worry about it. We agreed in meeting we were going to resolve this and other issues ogether if necessary. If you merge his PR first , we will resolve it in mine, otherwise in his. Just dont merge them at the same time.